--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -631,14 +631,16 @@
 			try {
 				fs.chmodSync(socketPath, 384);
 			} catch (error) {
-				bundlerDiagnostics.NUXT_B7018({ cause: error });
-				server.close();
-				try {
-					fs.rmSync(dirname(socketPath), {
-						recursive: true,
-						force: true
-					});
-				} catch {}
+				if (error.code !== "ENOENT") {
+					bundlerDiagnostics.NUXT_B7018({ cause: error });
+					server.close();
+					try {
+						fs.rmSync(dirname(socketPath), {
+							recursive: true,
+							force: true
+						});
+					} catch {}
+				}
 			}
 		});
 	} finally {
